import/simplex/include/NMObjective.h

```diff
@@ -6,7 +6,12 @@
 #if !defined _userfile_
 #define _userfile_
 
+#if defined(__APPLE__) || defined(__FreeBSD__)
+#include <stdlib.h>
+#else
 #include <malloc.h>
+#endif
+
```

Comment:
I'd say we can replace malloc with stdlib in the linux version as well
```suggestion
#include <stdlib.h>
```